### DISCUSSION TITLE
SB Obsrv - Release 5.2 & 5.3

### DISCUSSION THREAD

**Discussion by Shakthieshwari** [Sunbird-Obsrv/Community#51] [Query]
@rhwarrier Can you please let us know the release plan of Sunbird Obsrv Building Block for 5.2 and 5.3 release ?

Cc- @aishwaryashikshalokam @Prateek-slokam @aks30 @vijiurs

**Comment by Shakthieshwari** [comment]
@rhwarrier Any update on this please ?

Thanks

**Reply by Shakthieshwari** [reply]
Thanks @rhwarrier . Do Please keep us posted on this...

**Comment by rhwarrier** [comment]
This is being dicussed - we will shortly publish it for review and inputs as a new thread on the Obsrv discussion forum.

**Reply by Shakthieshwari** [reply]
@rhwarrier Any update on this please?

**Comment by rhwarrier** [comment]
This is work in progress, and should be published by the end of this week - if there are inputs your team has for the BB release, please start a new thread on the forum to discuss the same.

**Reply by Shakthieshwari** [reply]
Sure @rhwarrier

**Comment by rhwarrier** [comment]
The release details have been published here : https://github.com/Sunbird-Obsrv/Community/discussions/53
